recipes/recastnavigation/all/patches/001_fix_shared_option.patch

```diff
@@ -0,0 +1,129 @@
+--- CMakeLists.txt	2020-05-11 12:26:17.000000000 -0400
++++ CMakeLists.txt	2020-08-07 12:39:31.306787200 -0400
+@@ -9,7 +9,10 @@
+ option(RECASTNAVIGATION_DEMO "Build demo" ON)
+ option(RECASTNAVIGATION_TESTS "Build tests" ON)
+ option(RECASTNAVIGATION_EXAMPLES "Build examples" ON)
+-option(RECASTNAVIGATION_STATIC "Build static libraries" ON)
++
++if(MSVC AND BUILD_SHARED_LIBS)
++    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
++endif()
+
+ add_subdirectory(DebugUtils)
+ add_subdirectory(Detour)
+@@ -25,3 +28,4 @@
+     enable_testing()
+     add_subdirectory(Tests)
+ endif ()
++
+--- DebugUtils/CMakeLists.txt	2020-05-11 12:26:17.000000000 -0400
++++ DebugUtils/CMakeLists.txt	2020-08-07 12:40:03.921473100 -0400
+@@ -1,10 +1,6 @@
+ file(GLOB SOURCES Source/*.cpp)
+
+-if (RECASTNAVIGATION_STATIC)
```

Comment:
why do need to remove it if you can use RECASTNAVIGATION_STATIC as True/False?

Reply:
Conan already set `BUILD_SHARED_LIBS` as ON if the library must be shared.

I also made a PR removing `RECASTNAVIGATION_STATIC` https://github.com/recastnavigation/recastnavigation/pull/439

Reply:
Great, thanks for the additional PR.